configuration/debian/tedge_agent/postinst

```diff
@@ -25,6 +25,9 @@ fi
 # Create tedge-agent state directory /etc/tedge/.agent
 install -g tedge-agent -o tedge-agent -m 754 -d /etc/tedge/.agent
 
+# Create new operations file.
+install -g tedge -o tedge -m 754 -f /etc/tedge/operations/c8y/c8y_SoftwareUpdate
```

Comment:
I wonder if this the responsibility of the `tedge_agent` to tell what is supported by the `c8y-sm` mapper.